core/src/main/java/de/javakaffee/web/msm/LockingStrategy.java

```diff
@@ -497,7 +496,7 @@ public Void call() throws Exception {
             return null;
         }
 
-        public void saveSessionBackupFromResult( final BackupResult backupResult ) {
+        public void saveSessionBackupFromResult( final BackupResult backupResult ) throws IOException {
```

Comment:
I can't see it here, but probably it's now blocking while it shouldn't (at least not always).